etc/Makefile

```diff
@@ -65,7 +69,7 @@ define BUILD_KERNELSPEC
 $1: $$(FILE_$1)
 $$(FILE_$1): ../build/kernelspecs
 	rm -f $$(FILE_$1)
-	@( cd ../build/kernelspecs; tar -pvczf "../$$(FILE_$1)" $$(PATTERN_$1) )
+	@( cd ../build/kernelspecs; find . -name 'kernel.json' -print0 | xargs -0 sed -i.bak "s/VERSION/$(IMAGE_VERSION)/g"; find . -name *.bak -print0 | xargs -0 rm -f; tar -pvczf "../$$(FILE_$1)" $$(PATTERN_$1) )
```

Comment:
Please move the VERSION replacement to the `../build/kernelspecs` target (to the second to last statement just after the second cp command).  Otherwise we're performing the replacement multiple times when only the first attempt does anything and all other _massaging_ takes place above.  This target should be purely for applying the pattern to produce the various tar files (i.e., a read-only step).

Reply:
relocated sed replacement to ../build/kernelspecs target to avoid multiple iterations